composite/outlier-detection/src/config.ts

```diff
@@ -1,9 +1,9 @@
 import { util } from '@chainlink/ea-bootstrap'
-import types, { Execute } from '@chainlink/types'
+import { Execute } from '@chainlink/types'
 import { getSourceDataProviders, getSourceImpl } from './source'
 import { getCheckDataProviders, getCheckImpl } from './check'
 
-export type Config = types.Config & {
+export type Config = {
```

Comment:
@boxhock Changed this bit from your composite Outlier-Detection PR as it was throwing a TS error.

Just want to make sure this still works for you as expected. Opened #217 which includes investigating that this Config type here still works with the underlying adapter's Config.

I think we may want to verify that Configs are working as expected here while doing

Reply:
Yeah, it's not needed here.